engine/src/main/java/org/terasology/world/chunks/localChunkProvider/LocalChunkProvider.java

```diff
@@ -178,6 +191,22 @@ public ChunkViewCore getSubviewAroundChunk(Vector3i chunkPos) {
         return null;
     }
 
+    private ChunkViewCore createWorldView(BlockRegion region, Vector3i offset) {
+        Chunk[] chunks = new Chunk[region.getSizeX() * region.getSizeY() * region.getSizeZ()];
+        org.joml.Vector3i temp = new org.joml.Vector3i();
+        org.joml.Vector3i temp2 = new org.joml.Vector3i();
+        for (Vector3ic chunkPos : BlockRegionIterable.region(region).build()) {
+            Chunk chunk = chunkCache.get(JomlUtil.from(chunkPos));
+            if (chunk == null) {
+                return null;
+            }
+            temp.set(chunkPos).sub(region.getMinX(), region.getMinY(), region.getMinZ());
```

Comment:
This line doesn't actually do anything. It just changes `temp`, which is never read.